.github/workflows/publish.yml

```diff
@@ -12,48 +12,39 @@ on:
       - 'main'
 
 jobs:
-  docker-build-ghcr:
+  docker-build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: docker/metadata-action@v5
```

Comment:
it seems like we lose the tagged versions on GHCR/Docker Hub by removing this, right? Is there an example somewhere we can follow that accomplishes this correctly?

Reply:
`docker/metadata-action@v5` is actually still there, it's [here](https://github.com/protomaps/go-pmtiles/blob/9554132801055d0462df53a145553a0e0d560724/.github/workflows/publish.yml#L24), but done for both images. I tried to follow [this](https://docs.docker.com/build/ci/github-actions/push-multi-registries/) example and also tested it on my fork. It pushed to both dockerhub and ghcr.

Reply:
Not sure why the actions are not running on this PR, but we'll see, I'll tag a patch release. Thanks!

Reply:
(Oh, because it was a PR to a PR)